### PR TITLE
Add cloud symbols library

### PR DESCRIPTION
**Pull request by rfranzke**
Commonly used symbols for presentations related to cloud or cloud infrastructure.

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -427,5 +427,18 @@
...
+    "name": "Cloud",
+    "description": "Collection of symbols related to cloud artefacts.",
+    "authors": [
+      {
+        "name": "rfranzke",
+        "url": "https://twitter.com/rafaelfranzke"
+      }
+    ],
+    "source": "lipis/polygons.excalidrawlib",
+    "preview": "lipis/polygons.png",
```
Comment (lipis) [inline libraries.json:441]: update the paths

File: libraries.json [hunk 2]
```diff
@@ -427,5 +427,18 @@
...
+    "date": "2020-12-02"
```
Comment (lipis) [inline libraries.json:442]: Update the date

**Comment by dwelle** [comment]
@rfranzke sorry for the delay. Could you please add names of the services into the description for applicable items?

**Comment by rfranzke** [comment]
@dwelle done

File: libraries.json [hunk 3]
```diff
@@ -458,6 +458,19 @@
...
+    "description": "Collection of symbols related to cloud artefacts (logos of Kubernetes, Gardener, AWS, Azure, GCP and icons for illustrating architectures).",
+    "authors": [
+      {
+        "name": "rfranzke",
+        "url": "https://twitter.com/rafaelfranzke"
+      }
+    ],
+    "source": "cloud/cloud.excalidrawlib",
+    "preview": "cloud/cloud.png",
+    "date": "2021-04-21"
```
Comment (dwelle) [inline libraries.json:472]:
```suggestion
    "date": "2021-06-07"
```

Comment (dwelle) [reply libraries.json:472]: cannot commit as you must have forgot to `[x] allow maintained to edit`

Comment (rfranzke) [reply libraries.json]: done

**Comment by dwelle** [comment]
Thanks @rfranzke!

**Pull request merged**